full-moon/src/ast/types.rs

```diff
@@ -302,3 +302,70 @@ impl<'a> TypeSpecifier<'a> {
         &self.type_info
     }
 }
+
+macro_rules! make_op {
```

Comment:
If this is C&P from where we make BinOp and UnOp, this should be in a utility module somewhere instead.

Reply:
I’m a bit unsure on this. I tried using #[macro_export] above the same code in the original module, then #[macro_use] above the import in this file, and it seemed to error. I’ll dig into this more, and probably move it into a utility file

Reply:
Importing macros is kinda funky, but we define them outside for things like `define_parser!`

Reply:
I have moved this into the `parser_utils` module - although I don't know if I should create a separate utility module instead as it isn't *specifically* for the parser